prow/cmd/deck/main.go

```diff
@@ -462,6 +464,31 @@ func handleBadge(ja *jobs.JobAgent) http.HandlerFunc {
 	}
 }
 
+func handleJobHistory(templateRoot string, ca *config.Agent, gcsClient *storage.Client) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		tmpl, err := getJobHistory(r.URL, ca.Config(), gcsClient)
+		if err != nil {
+			msg := fmt.Sprintf("Failed to get job history: %v", err)
+			logrus.Error(msg)
```

Comment:
Please use `WithField` to add more context to this logged error. Specifically, the request URL should be included.